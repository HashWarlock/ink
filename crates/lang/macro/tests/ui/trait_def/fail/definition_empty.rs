use ink_lang as ink;

#[ink::trait_definition]
pub trait EmptyDefinition {}

fn main() {}
