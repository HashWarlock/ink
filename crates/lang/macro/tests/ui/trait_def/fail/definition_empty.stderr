error: encountered invalid empty ink! trait definition
 --> $DIR/definition_empty.rs:4:1
  |
4 | pub trait TraitDefinition {}
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
